airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/metadata.py

```diff
@@ -149,8 +149,33 @@ def _run(self, connector: Connector) -> CheckResult:
         )
 
 
+class CheckConnectorMaxSecondsBetweenMessagesValue(MetadataCheck):
```

Comment:
Does this check only applies to sources?
If it does please set:
`applies_to_connector_types = ["source"]`

Reply:
Done below